alembic/versions/2024_04_08_0395f682958e_add_order_delivery.py

```diff
@@ -0,0 +1,24 @@
+"""Add order delivery
+
+Revision ID: 0395f682958e
+Revises: b614a52759c5
+Create Date: 2024-04-08 16:35:41.629048
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "0395f682958e"
+down_revision = "b614a52759c5"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column("order", sa.Column("delivered", sa.Boolean(), nullable=False))
```

Comment:
Slight preference for `is_delivered` but go with what you like the most.

Reply:
Renamed in https://github.com/Clinical-Genomics/cg/pull/3095/commits/017bc5dcc43cc1005f405e623c40454b8ae8e587